lib/red_cross/resque/plugins/red_cross_monitor.rb

```diff
@@ -0,0 +1,66 @@
+module Resque
+  module Plugins
+    module RedCrossMonitor
+      # Hash of properties for RedCross.monitor
+      # Please add count: MY_NUMBER in case your metric value differs 1
+      def job_method(*args)
+        #Add the method that triggered by the job
```

Comment:
this should return
```
@job_method
```
generally you should define an attribute accessor for job_method